os/constants-and-ids.md

```diff
@@ -0,0 +1,23 @@
+# Constants and IDs
+
+This document contains well-known constants and IDs used by Container Linux.
+
+## Omaha application ID
+
+This UUID is used to identify Container Linux to the update service, i.e. as an `appid` over the [Omaha protocol](../coreupdate/update-protocol.md).
+
+| Label            | Value                                  | Notes |
+|------------------|----------------------------------------|-------|
+| Container Linux  | `e96281a6-d1af-4bde-9a0a-97b76e56dc57` | -     |
+
+## GPT partition types
+
+These GUIDs are dedicated [GPT partition types](https://en.wikipedia.org/wiki/GUID_Partition_Table#Partition_type_GUIDs) for specific Container Linux usages.
+
+| Label              | Value                                  | Notes |
+|--------------------|----------------------------------------|-------|
+| `coreos-usr`       | `5dfbf5f4-2848-4bac-aa5e-0d9a20b745a6` | Alias for historical `coreos-rootfs`, currently used for `/usr` only |
+| `coreos-resize`    | `3884dd41-8582-4404-b9a8-e9b84f2df50e` | Support for auto-resizing via `extend-filesystems`, current default type for `/` |
+| `coreos-reserved`  | `c95dc21a-df0e-4340-8d7b-26cbfa9a03e0` | Reserved for future usage |
```

Comment:
It turned out this is actually already in use for `OEM-CONFIG` partition, I'll update the note.